modules/develop/pages/integrations.adoc

```diff
@@ -0,0 +1,343 @@
+a|
+* https://docs.couchbase.com/server/current/connectors/odbc-jdbc-drivers.html[Couchbase ODBC and JDBC Drivers]
+* https://www.cdata.com/drivers/couchbase/docs/[CData Driver]
+* https://www.cdata.com/drivers/couchbase/#section-etl-anchor[Popular Data Warehousing Integrations]
+* https://www.cdata.com/drivers/couchbase/#section-data-anchor[Data Management Tools]
+* https://www.cdata.com/drivers/couchbase/#section-dev-anchor[Developer Tools]
+* https://www.cdata.com/drivers/couchbase/#section-bi-anchor[BI & Data Visualization]
+| ✔
+| ✔
+| Partner Supported
+|===
+
+
+
+== IDE Integrations
+
+
+.IDE Plugins
+[cols="20,45,50,20,20,35"]
+|===
+| Integration | Summary | Links | Capella | Self-managed |  <<#support-model,Support Model>>
+
+| VS Code extension for Couchbase
+| A seamless experience for Couchbase Server users who want to work within the popular Visual Studio Code editor.
+a|
+* https://marketplace.visualstudio.com/items?itemName=Couchbase.vscode-couchbase[Visual Studio Marketplace]
+* https://github.com/couchbaselabs/VSCode-Couchbase/blob/main/GETTING-STARTED.md[Tutorial]
+* https://github.com/couchbaselabs/VSCode-Couchbase/blob/main/SNIPPETS.md[Code snippets]
+| ✔
+| ✔ 
+| Couchbase Community Support
+
+
+| VS Code extension for Couchbase Lite
+| A seamless experience for Couchbase Mobile users who want to work within the popular Visual Studio Code editor.
+a| 
+* https://marketplace.visualstudio.com/items?itemName=Couchbase.vscode-cblite[Visual Studio Marketplace]
+* https://github.com/couchbaselabs/vscode-cblite[GitHub repo and intro]
+| ✔
+| ✔ 
+| Couchbase Community Support
+
+
+| JetBrains
+| Provides support for Couchbase in JetBrains IDEs --  to develop Couchbase applications, complete with code completion, syntax highlighting, and debugging support.
+a|
+* https://plugins.jetbrains.com/plugin/22131-couchbase[JetBrains Marketplace]
+// * [Tutorial]
+* https://github.com/couchbaselabs/couchbase_jetbrains_plugin[GitHub source repo]
+| ✔
+| ✔
+| Couchbase Community Support
+|===
+
+
+
+== Data Tools
+
+Import, export, and query data from the command line.
+
+.Data Tools
+[cols="20,45,50,20,20,35"]
+|===
+| Integration | Summary | Links | Capella | Self-managed |  <<#support-model,Support Model>>
+
+
+| Couchbase Shell (cbsh)
+| Interactive shell, based upon nushell, which you can use to extract, transform, and load different data formats in Couchbase.
+a| 
+* https://couchbase.sh/docs/[cbsh docs]
+* https://github.com/couchbaselabs/couchbase-shell[GitHub repo]
+| ✔
+| ✔ 
+| Couchbase Officially Supported (currently in _Developer Preview_)
+
+
+| cbimport
+| Utilitly for importing data into Couchbase databases.
+Included in self-managed Couchbase Server, or downloadable as part of the xref:cloud:reference:command-line-tools.adoc#download-and-install-the-couchbase-command-line-tools[Capella Tools package].
+a| xref:server:tools:cbimport.adoc[cbimport reference]
```

Comment:
Just checking &mdash; is the `cbimport` documentation available in the cloud docs as well? If so, remember to link to the documentation on the cloud side when porting this PR to the cloud branches.

Reply:
Do you think the link in the previous cell is enough?

Reply:
Fine for now!